lib/pages/instance.dart

```diff
@@ -0,0 +1,436 @@
+                                  decoration: BoxDecoration(
+                                    shape: BoxShape.circle,
+                                    image: DecorationImage(
+                                      fit: BoxFit.cover,
+                                      image: imageProvider,
+                                    ),
+                                  ),
+                                ))
+                        : SizedBox(width: 50),
+                  ))
+            else if (commSnap.hasError)
+              Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: Text("Can't load communities, ${commSnap.error}"),
+              )
+            else
+              Padding(
+                padding: const EdgeInsets.symmetric(vertical: 10),
+                child: CircularProgressIndicator(),
+              ),
+            ListTile(
+              title: Center(child: Text('See all')),
+              onTap: goToCommunities,
+            ),
+            _Divider(),
+            ListTile(
+              title: Center(
+                child: Text(
+                  'Admins:',
+                  style: theme.textTheme.headline6.copyWith(fontSize: 18),
+                ),
+              ),
+            ),
+            ...site.admins.map((e) => ListTile(
+                  title: Text(e.preferredUsername ?? '@${e.name}'),
+                  subtitle: e.bio != null ? MarkdownText('${e.bio}') : null,
+                  onTap: () => goToUser(e.id),
+                  leading: e.avatar != null
+                      ? CachedNetworkImage(
+                          height: 50,
+                          width: 50,
+                          imageUrl: e.avatar,
+                          imageBuilder: (context, imageProvider) => Container(
+                                decoration: BoxDecoration(
+                                  shape: BoxShape.circle,
+                                  image: DecorationImage(
+                                      fit: BoxFit.cover, image: imageProvider),
+                                ),
+                              ))
+                      : SizedBox(width: 50),
+                )),
+            _Divider(),
+            ListTile(
+              title: Center(
+                child: Text(
+                  'Recently banned:',
+                  style: theme.textTheme.headline6.copyWith(fontSize: 18),
+                ),
+              ),
+            ),
+            ...site.banned.reversed.take(5).map((e) => ListTile(
+                  // subtitle: Text(),
+                  title: Text(
+                      (e.preferredUsername == null || e.preferredUsername == '')
+                          ? '@${e.name}'
+                          : e.preferredUsername),
+                  onTap: () => goToUser(e.id),
+                  leading: SizedBox(width: 50),
+                )),
```

Comment:
Maybe we shouldn't show recently banned and just leave a button "show banned users"